pinot-controller/src/main/resources/static/js/lib/codemirror/hint/sql-hint.js

```diff
@@ -0,0 +1,322 @@
+/*
```

Comment:
remove apache header

Reply:
Ok, I don't see through because there is no checks apache header, so direct copy apache header from the controller/src/main/resources/static/css/lib/codemirror-hint.css and controller/src/main/resources/static/js/init.js. Now, I remove apache header